src/test/java/com/connexta/ingest/IngestITests.java

```diff
@@ -244,6 +244,7 @@ public void testSuccessfulIngestRequest() throws Exception {
         .header(
             "Accept-Version",
             "0.5.0") // TODO inject this like we do for the transformApiVersion in TransformClient
+        .header("Last-Modified", "2017-06-11T14:32:28Z")
```

Comment:
❓Why are all of the timestamps different in this class? Would testing different timestamps be more appropriate as a parametrized test?

Reply:
Exactly, will update